apps/kibana/templates/services/app/configs/kibana.yml.j2

```diff
@@ -15,10 +17,13 @@ server.publicBaseUrl: {{ kibana_server_public_base_url | default("https://" + ki
 elasticsearch.hosts: {{ kibana_es_hosts | to_yaml }}
 
 {% if kibana_security_enabled %}
-elasticsearch.username: "${KIBANA_ELASTIC_USERNAME}"
-elasticsearch.password: "${KIBANA_ELASTIC_PASSWORD}"
+elasticsearch.username: ${KIBANA_ELASTIC_USERNAME}
+elasticsearch.password: ${KIBANA_ELASTIC_PASSWORD}
 elasticsearch.ssl.certificateAuthorities: ["{{ kibana_ca_path }}"]
-xpack.security.encryptionKey: "${XPACK_SECURITY_ENCRYPTIONKEY}"
+xpack.security.encryptionKey: ${XPACK_SECURITY_ENCRYPTIONKEY}
+xpack.reporting.encryptionKey: ${XPACK_REPORTING_ENCRYPTIONKEY}
+xpack.encryptedSavedObjects.encryptionKey: ${XPACK_ENCRYPTEDSAVEDOBJECTS_ENCRYPTIONKEY}
+xpack.encryptedSavedObjects.keyRotation.decryptionOnlyKeys: ${XPACK_ENCRYPTEDSAVEDOBJECTS_KEYROTATION_DECRYPTIONONLYKEYS}
```

Comment:
`XPACK_ENCRYPTEDSAVEDOBJECTS_KEYROTATION_DECRYPTIONONLYKEYS` is hard to read... it is not possible to split `DECRYPTIONONLYKEYS` ?

Reply:
In fact no as there is an automated mapping between environment variables and kibana configuration variables names (underscores became dots).